### ISSUE TITLE
Reactive form setting editor disabled has no effect before editor loaded

### ISSUE THREAD

**Issue by pbarranis** [tinymce/tinymce-angular#56]
I have an Angular 7 reactive form which sets the editor to disabled before the page has a chance to finish completely loading.  My code fetches the record being viewed on-screen, but it's on my dev machine, so the response time is very fast (likely before loading and rendering has finished).

I did a quick test to see if I placed on button the screen and disabled the editor using the exact same line, but in this case I was testing after the component had fully rendered.  It worked fine.  So I know the problem is strictly a before-fully-rendered issue.

I looked at the source and think I spotted the issue: [line 83](https://github.com/tinymce/tinymce-angular/blob/90b7c602a6b5f5b1309ccddad083f4210bce1f84/tinymce-angular-component/src/editor/editor.component.ts#L83) sets `readonly: true` in the init.  I can't find any documentation on the TinyMCE website that says that that's a valid init option; am I overlooking something?

I'm running version 2.3.2 of this package on 7.1.3 of Angular.  Due to the way this package loads TinyMCE from the cloud, I don't know how to find what version number of TinyMCE I'm running, but I'm happy to provide it if someone can tell me where to look.

**Comment by ronfogel** [comment]
I am using formControlName and its not working for me at all. is it working for you? using same versions like yours..

**Comment by pbarranis** [comment]
Following the approach of using a [disabled] binding, but I get the browser alert in the console mentioned [in this post.](https://stackoverflow.com/questions/40494968/reactive-forms-disabled-attribute)  So I tried using the attr.disabled solution mentioned [in this answer,](https://stackoverflow.com/a/43765804/358578) but it has no effect at all for the editor element.  Note that it works fine on other elements, like inputs, so it's a specific issue with the editor element.

Using [disabled] seems to be an acceptable workaround, but I'd really like a solution that doesn't involve getting that ugly warning in the console.

**Comment by fyrkant** [comment]
`readonly` is a valid init option, though it might not be documented, hmm.

So the warning about using the `disabled` attribute only comes when you are using the editor inside of a reactive form?

**Comment by pbarranis** [comment]
@fyrkant Oh good to hear from someone!  I was going to contact support today and ask if someone could please look at this issue.

Yes.  I believe the warning is from zone.js checking for Angular worst-practices.  I had the [disabled] binding on at least one other place in my code, but for some reason I only get the warning where I use TinyMCE.

Of course the root problem I want to solve is why calling .disable() on the Angular Control object isn't disabling the control with the timing issues mentioned in the OP.  Any help would be greatly appreciated.

**Comment by pbarranis** [comment]
I just updated all my packages, and the problem has gone away.  I updated Angular from 7.1.3 to 7.2.0 and @tinymce/tinymce-angular from 2.3.3 to 2.4.0.  I skimmed through the commits to tinymce-angular since 2.3.3, and I don't see how any of them would have solved the problem.  That said, fixed is fixed.

I'm closing this by virtue of No Repro.  If the issue crops up again, it can always be reopened.
